feat: API (merkle, envio) and App (airstreams) updates (#130)
sablier-labs/v2-docs: docs/api/indexers/01-endpoints.mdx

---
id: "endpoints"
sidebar_position: 1
title: "Endpoints"
---

import LinkPreview from "@site/src/components/LinkPreview";

:::tip Chains: Individual vs. Aggregated

> While subgraphs have a chain-focused approach (one subgraph endpoint/deployment for every chain), it is important to

Reviewer: Do you mean "one subgraph endpoint/deployment for every chain" or "one subgraph endpoint/deployment per chain"?

> note that Envio will index all chains within the same deployment. This makes is so data from all chains is available to

Reviewer: ```suggestion
note that Envio will index all chains within the same deployment. This makes it so data from all chains can be queried from the same place
```
